Reneiski_Homework1/ReneiskiSelenium11/Program.cs

```diff
@@ -0,0 +1,246 @@
+
+            Assert.IsTrue(currentResultText1.Contains("You have selected "));
+            Assert.IsTrue(currentResultText1.Contains("Impressive"));
+        }
+
+        [Test]
+        public void ThirdWebTablesTest()
+        {
+            _driver.Navigate().GoToUrl("https://demoqa.com/webtables");
+
+            var currentResult = _driver.FindElement(By.XPath("//*[@class='rt-table']"));
+            var currentResultText = currentResult.Text;
+
+            Assert.IsTrue(currentResultText.Contains("Cierra"));
+            Assert.IsTrue(currentResultText.Contains("Vega"));
+            Assert.IsTrue(currentResultText.Contains("39"));
+            Assert.IsTrue(currentResultText.Contains("cierra@example.com"));
+            Assert.IsTrue(currentResultText.Contains("10000"));
+
+            var addButton = _driver.FindElement(By.XPath("//*[@id='addNewRecordButton']"));
+            addButton.Click();
+
+            var firstName = "Alex";
+            var firstNameTextBox = _driver.FindElement(By.XPath("//*[@id='firstName']"));
+            firstNameTextBox.SendKeys(firstName);
+
+            var lastName = "Ren";
+            var lastNameTextBox = _driver.FindElement(By.XPath("//*[@id='lastName']"));
+            lastNameTextBox.SendKeys(lastName);
+
+            var userEmail = "Ren@tut.by";
+            var userEmailTextBox = _driver.FindElement(By.XPath("//*[@id='userEmail']"));
+            userEmailTextBox.SendKeys(userEmail);
+
+            var userAge = "41";
+            var userAgeTextBox = _driver.FindElement(By.XPath("//*[@id='age']"));
+            userAgeTextBox.SendKeys(userAge);
+
+            var userSalary = "20000";
+            var userSalaryTextBox = _driver.FindElement(By.XPath("//*[@id='salary']"));
+            userSalaryTextBox.SendKeys(userSalary);
+
+            var userDepartment = "QA";
+            var userDepartmentTextBox = _driver.FindElement(By.XPath("//*[@id='department']"));
+            userDepartmentTextBox.SendKeys(userDepartment);
+
+            var submitButton = _driver.FindElement(By.XPath("//*[@id='submit']"));
+            submitButton.Click();
+
+            var currentResult1 = _driver.FindElement(By.XPath("//*[@class='rt-table']"));
+            var currentResultText1 = currentResult1.Text;
+
+            Assert.IsTrue(currentResultText1.Contains(firstName));
+            Assert.IsTrue(currentResultText1.Contains(lastName));
+            Assert.IsTrue(currentResultText1.Contains(userAge));
+            Assert.IsTrue(currentResultText1.Contains(userEmail));
+            Assert.IsTrue(currentResultText1.Contains(userSalary));
+            Assert.IsTrue(currentResultText1.Contains(userDepartment));
+
+            var editButton = _driver.FindElement(By.XPath("//*[@id='edit-record-4']"));
+            editButton.Click();
+
+            var editedUserSalary = "22000";
+            var editedUserSalaryTextBox = _driver.FindElement(By.XPath("//*[@id='salary']"));
```

Comment:
you have already created userSalaryTextBox element that has the same locator, you can use it
the same for another elements below

Reply:
Fixed, but I wasn't able to use userSalaryTextBox element because the modal for editing was reopened for this action